### ISSUE TITLE
Saving and Reloading Adapters - AdapterFusion

### ISSUE THREAD

**Issue by nehalelkaref** [adapter-hub/adapter-transformers#477]
Hello.
I have trained an AdapterFusion model (with BERT being the base model) then saved the fusion layers adapter and all other pre-trained adapters I have used in that fusion layer and my prediction head. When reloading the pretrained adapters and the adapter fusion into my base model later once again and try to evaluate my model on a dev set, I get a huge performance gap. So I want to make sure if my saving and reloading steps are correct and if there is something I am not saving/reloading, please?

To save the adapters and model head:
```
model.save_head()
model.save_all_adapter_fusions("./path")
model.save_adapter_fusion("./path", model.active_adapters) #model.active_adapters yields the one and only trained fusion adapter
```

To load adapters into the model once again:
```
model.load_adapter("./path", overwrite_ok=True, with_head=False)
model.load_adapter_fusion("./path", with_head=True, set_active=True)
```

**Comment by adapter-hub-bert** [comment]
This issue has been automatically marked as stale because it has been without activity for 90 days. This issue will be closed in 14 days unless you comment or remove the stale label.

**Comment by JiahuiSophieHU** [comment]
Hello @nehalelkaref , 
I met the same problem with my trained adapters fusion model for text classification (a drop of 5% of F1 score). Could you let me know if you found a solution to this problem? 
I tried to save with the following methods, but both failed. 
And I load models as you did by trying different combinations of overwrite_ok, set_active, with_head parameters.
Method 1:
model.save_head(path)
model.save_adapter_fusion(path, "mrpc,record,quail,newsqa")
model.save_all_adapters(path)
Method 2:
trainer.save_model(path)
Thanks
